manifests/reports.pp

```diff
@@ -15,27 +15,30 @@
 #
 class datadog_agent::reports(
   $api_key,
+  $manage_ruby,
```

Comment:
I think this should be set to `true` by default to avoid breaking manifests not accounting for the option.